library/chronograf

```diff
@@ -0,0 +1,8 @@
+maintainer: Shubhra Kar <shubhra@influxdb.com> (@ShubhraKar)
```

Comment:
I've got a few PR's to fix this. One of us will update this PR when we fix this.